src/objectExplorer/objectExplorerUtils.ts

```diff
@@ -15,8 +16,13 @@ export class ObjectExplorerUtils {
             return path.join(ObjectExplorerUtils.rootPath, `${label}.svg`);
         }
     }
-
     public static getNodeUri(node: TreeNodeInfo): string {
```

Comment:
>getNodeUri [](start = 18, length = 10)

Your changes aren't really doing this anymore though - you're always getting the Server node URI. Why is this change necessary?

Reply:
https://github.com/microsoft/vscode-mssql/pull/1301/files#diff-9401975d07f4df26ce1df2d776b8c746R277